homeassistant/components/zwave.py

```diff
@@ -209,39 +209,37 @@ def _obj_to_dict(obj):
             if key[0] != '_' and not hasattr(getattr(obj, key), '__call__')}
 
 
-def _node_name(node):
+def node_name(node):
     """Return the name of the node."""
     return node.name or "{} {}".format(
         node.manufacturer_name, node.product_name)
 
 
-def _value_name(value):
+def value_name(value):
     """Return the name of the value."""
-    return "{} {}".format(_node_name(value.node), value.label)
+    return "{} {}".format(node_name(value.node), value.label)
 
 
-def _node_object_id(node):
+def node_object_id(node):
```

Comment:
I actually believe ff this is changed to include the value.index We can make one sensor for each index. Though this will be a breaking change, as all zwave entities will include the index number. And of course it needs to be tested.